adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java

```diff
@@ -79,20 +89,13 @@
 
     private Intent mIntentToStartActivity;
 
-    private AuthenticationActivity mActivity;
-
-    public AuthenticationActivityUnitTest() {
-        super(AuthenticationActivity.class);
-    }
+    @Rule
+    public ActivityTestRule<AuthenticationActivity> mActivityRule = new ActivityTestRule(AuthenticationActivity.class, true, false);
```

Comment:
It looks like this field can be made `final` -- I'm also seeing a warning in my IDE for using the raw-type constructor which can be suppressed by replacing this line with:
`    public final ActivityTestRule<AuthenticationActivity> mActivityRule = new ActivityTestRule<>(AuthenticationActivity.class, true, false);`